doc/1.4/language.md

```diff
@@ -3466,7 +3466,8 @@ after <em>scalar</em> <em>unit</em>: <em>method</em>(<em>e1</em>, ... <em>eN</em
 In this form, the specified point in the future is given through a time delay
 (in simulated time, measured in the specified time unit) relative to the time
 when the after delay statement is executed. The currently supported time units
-are `s` for seconds and `cycles` for cycles.
+are `s` for seconds (fractional), `ps` for picoseconds (integer),
```

Comment:
Although the term 'fractional' is a pretty good word here, we don't use it elsewhere in the document. Perhaps "floating-point" is better?

Wording is somewhat tricky in this domain because the true technical term is the hopelessly ambiguous word 'double'.

Reply:
"Fractional numbers" is a pretty standard way to refer to, well, fractional numbers. I think using "fractional" is fine.

Reply:
Upon further consideration, I would do
> are `s` for seconds (`double`), `ps` for picoseconds (`uint64`), `cycles` for cycles (`uint64`)

Agreed?

Reply:
maybe
> (with type `double`)
?

Reply:
> "Fractional numbers" is a pretty standard way to refer to, well, fractional numbers. I think using "fractional" is fine.

I agree that fractional is a sane word here in principle, but it is inconsistent with the rest of our docs. E.g. we say things like "Floating-point numbers: There is only one floating-point type, called `double`. It corresponds to the C type `double`.". I would be OK with a global `s/floating-point/fractional` in our docs, but mixing the two adds unnecessary confusion.

I'm also OK with "type `double`", or just floating-point, if you prefer that.